Cargo.toml

```diff
@@ -23,7 +25,7 @@ rusoto_core = ">=0.32.0"
 rusoto_credential = ">=0.11.0"
 rusoto_ses = ">=0.32.0"
 rusoto_sqs = ">=0.32.0"
-sendgrid = ">=0.7.0"
+sendgrid = { git = "https://github.com/gsquire/sendgrid-rs.git" }
```

Comment:
0.7.1 is already available with your patch https://crates.io/crates/sendgrid/0.7.1 right?

Reply:
ahh yes that's right :)